.circleci/config.yml

```diff
@@ -357,6 +340,7 @@ jobs:
         default: build
       image:
         type: string
+        default: docker
```

Comment:
This probably isn't necessary since we're not specifying a default for either of the build jobs and we're passing this parameter from every job that calls it.